Bottom: 4758995328e901f0db5273b5faac96592df5db5c
Top:    ede31a24133156cd778b88c9c3c8385a3554590f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2014-01-02 21:45:30 -0500

Refresh of zarlink

---

diff --git a/target/linux/ramips/rt305x/profiles/asiarf.mk b/target/linux/ramips/rt305x/profiles/asiarf.mk
index 111cd80..34f61cc 100644
--- a/target/linux/ramips/rt305x/profiles/asiarf.mk
+++ b/target/linux/ramips/rt305x/profiles/asiarf.mk
@@ -7,7 +7,7 @@
 
 define Profile/AWM002EVB
 	NAME:=AsiaRF AWM002-EVB
-	PACKAGES+=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
+	PACKAGES:=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
 	kmod-i2c-core kmod-i2c-gpio
 endef
 
@@ -19,9 +19,9 @@ $(eval $(call Profile,AWM002EVB))
 
 define Profile/ZARLINK
 	NAME:=AsiaRF Zarlink
-	PACKAGES+=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
-	kmod-sound-core \
-	kmod-i2c-core kmod-i2c-gpio kmod-sound-soc-rt5350-le89156
+	PACKAGES:=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
+	kmod-i2c-core kmod-i2c-gpio  \
+	kmod-sound-core kmod-sound-soc-rt5350-le89156
 endef
 
 define Profile/ZARLINK/Description
